From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Tue, 6 Dec 2016 11:01:22 -0500
Subject: agent: Respect --enable-large-secmem

* agent/gpg-agent.c (main): Initialize secmem to the configured buffer
size.

--

This patch is a step toward addressing
GnuPG-bug-id: 2857

Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
(cherry picked from commit 8636ad5023a1bdc527add40a5508f8c5b7c35221)
---
 agent/gpg-agent.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/agent/gpg-agent.c b/agent/gpg-agent.c
index efef8c1..196d6ac 100644
--- a/agent/gpg-agent.c
+++ b/agent/gpg-agent.c
@@ -1066,7 +1066,7 @@ main (int argc, char **argv )
     }
 
   /* Initialize the secure memory. */
-  gcry_control (GCRYCTL_INIT_SECMEM, 32768, 0);
+  gcry_control (GCRYCTL_INIT_SECMEM, SECMEM_BUFFER_SIZE, 0);
   maybe_setuid = 0;
 
   /*
